cmd/launcher/doctor.go

```diff
@@ -13,6 +13,9 @@ import (
 )
 
 func runDoctor(args []string) error {
+	attachConsole()
+	defer detachConsole()
```

Comment:
I think we should do this everywhere but desktop. I'm not sure if it's easier to put this into main with an exception for dekstop, or in each piece

Reply:
Looking through --

* I can add it to `flare` -- that gets us "Flare uploaded successfully" + the file name, so that seems useful
* Adding it to `interactive` breaks interactive
* Adding it to `version` makes sense
* I'm not sure why we'd want it for launcher proper stuff (runLauncher, windows service, windows service foreground, extension) and I am a little nervous that adding it there would break something
* `uninstall` isn't implemented on Windows so it doesn't hurt to add it there

Reply:
I think we want it in `runLauncher` -- that's mostly going to be used by developers in a console window

`windows service foreground` is an interesting one. That's designed to run in the foreground to debug issues, it'll never run through the service manager. Probably needs it

Reply:
`runLauncher` is called by the Windows service and that's why I'm hesitant to add it there -- I'll try it out and see how it behaves.

I'll try adding it to foreground and see how it behaves.

Reply:
It works fine in foreground and I've added it there.

It also seems fine for `runLauncher`, tentatively, but I'm really not sure we gain a lot there for the risk that it affects something in an unexpected way.

Reply:
Hrm. I guess that's a good question. I think there's value in getting console output for developers. But it's hard to quantify the risk. We used to run launcher without windowsgui, and that was fine. But this is different. I guess we can go either way

Reply:
I agree there's value in getting console output for developers -- I think my preference is to not add it for `runLauncher` now to avoid unnecessary risk on Windows since [this issue](https://github.com/kolide/launcher/issues/1387) is on my mind (knowing we're thinking of solving the output issue more cleanly and thoroughly in the future with a separate desktop binary).

Reply:
Okay. Changing it is the status quo, so let's kick it forward.